int64 time_stamp

string[] link_name
int32[] robot_num
std_msgs/Float32MultiArray position
std_msgs/Float32MultiArray quaternion